routes/user.js

```diff
@@ -163,6 +164,32 @@ router.get(
   }),
 );
 
+//GET all user conversations PRIVATE
+router.get('/:id/conversations', auth, async (req, res) => {
+  const userId = req.params.id;
```

Comment:
If this is a private route, we can make it just '/conversations' and get the user information from our auth middleware rather than using an id parameter